pontoon/base/views.py

```diff
@@ -1064,9 +1064,8 @@ def amagama(request):
         return HttpResponse("error")
 
 
-def transvision(request, repo, title):
+def transvision(request):
     """Get Mozilla translations from Transvision service."""
-    log.debug("Get Mozilla translations from Transvision service.")
```

Comment:
This totally works. A future improvement would be to abstract machinery services on the client side to some API that each services implements, and then have the Transvision one contact Transvision directly.